packages/test-base/src/androidAndroidTest/kotlin/io/realm/kotlin/test/shared/RealmDictionaryTests.kt

```diff
@@ -1503,9 +1501,9 @@ internal abstract class ManagedDictionaryTester<T>(
     }
 
     override fun entries_iteratorConcurrentModification() {
-        // Ignore ByteArray and RealmObject: structural equality cannot be assessed for these types
-        // when removing entries from the entry set
-        if (classifier != ByteArray::class && classifier != RealmObject::class) {
+        // Ignore RealmObject: structural equality cannot be assessed for this type when removing
+        // entries from the entry set
+        if (classifier != RealmObject::class) {
```

Comment:
Maybe I'm missing it, but are we testing removing objects anywhere?

Reply:
After talking. We should add a TODO linking to https://github.com/realm/realm-kotlin/issues/1097